com.unity.render-pipelines.core/ShaderLibrary/ACES.hlsl

```diff
@@ -501,16 +507,19 @@ half segmented_spline_c5_fwd(half x)
     return pow(10.0, logy);
 }
 
-half segmented_spline_c9_fwd(half x)
+struct SegmentedSplineParams_c9
 {
-    const half coefsLow[10] = { -1.6989700043, -1.6989700043, -1.4779000000, -1.2291000000, -0.8648000000, -0.4480000000, 0.0051800000, 0.4511080334, 0.9113744414, 0.9113744414 }; // coefs for B-spline between minPoint and midPoint (units of log luminance)
-    const half coefsHigh[10] = { 0.5154386965, 0.8470437783, 1.1358000000, 1.3802000000, 1.5197000000, 1.5985000000, 1.6467000000, 1.6746091357, 1.6878733390, 1.6878733390 }; // coefs for B-spline between midPoint and maxPoint (units of log luminance)
-    const half2 minPoint = half2(segmented_spline_c5_fwd(0.18 * exp2(-6.5)), 0.02); // {luminance, luminance} linear extension below this
-    const half2 midPoint = half2(segmented_spline_c5_fwd(0.18), 4.8); // {luminance, luminance}
-    const half2 maxPoint = half2(segmented_spline_c5_fwd(0.18 * exp2(6.5)), 48.0); // {luminance, luminance} linear extension above this
-    const half slopeLow = 0.0; // log-log slope of low linear extension
-    const half slopeHigh = 0.04; // log-log slope of high linear extension
+    float coefsLow[10];    // coefs for B-spline between minPoint and midPoint (units of log luminance)
+    float coefsHigh[10];   // coefs for B-spline between midPoint and maxPoint (units of log luminance)
+    half2 minPoint; // {luminance, luminance} linear extension below this
+    half2 midPoint; // {luminance, luminance}
+    half2 maxPoint; // {luminance, luminance} linear extension above this
+    float slopeLow;       // log-log slope of low linear extension
+    float slopeHigh;      // log-log slope of high linear extension
+};
 
+half segmented_spline_c9_fwd(half x, SegmentedSplineParams_c9 params)
```

Comment:
Note: we will need to mention this change in upgrade guide

Reply:
Yeah to be safe I added the old signature/version so it is compatible and should work like before